src/Optimization/hiopIterate.cpp

```diff
@@ -396,67 +408,122 @@ bool hiopIterate::updateDualsIneq(const hiopIterate& iter, const hiopIterate& di
 }
 */
 
-int hiopIterate::adjust_small_slacks(hiopVector& slack, 
-                                     const hiopVector& bound, 
-                                     const hiopVector& slack_dual, 
-                                     const hiopVector& select,
-                                     const double& mu)
+int hiopIterate::adjust_small_var_slacks(hiopVector& slack,
```

Comment:
I liked it better before, when was only one `adjust_small_slacks_vars` or similarly named method. The functionality of the two news methods is identical. I suggest to go back to one function and pass the cached pointers as arguments to the method, would that be too much hassle?